IFC 4.3/UT_Alignment_12d_3/readme.md

```diff
@@ -0,0 +1,54 @@
+
+# UT_Alignment_12d_3
+# Simple alignment with left arc in horizontal geometry and crest parabola for vertical geometry
+# There is only the semantic definition and no ifc geometry 
+
+### Intent
+
+IFC 4x3 RC2
+This scenario has a simple alignment consisting of a:
+<<<<<<< Updated upstream
```

Comment:
It seems like the readme-file needs to be re-uploaded. Same issue as with the IFC files.